src/modules/input.wren

```diff
@@ -115,6 +115,7 @@ class Mouse {
   foreign static hidden=(value)
   foreign static relative=(value)
   foreign static relative
+  foreign static setCursor(name)
```

Comment:
I agree. After working with the method on a practical case, It it makes sense to have a getter. It would be better if it used that signature as well. I'll look into implementing that.

Reply:
fixed in a028707